api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.5.x.xml

```diff
@@ -390,4 +390,102 @@
 		              tableName="encounter_diagnosis" />
 	</changeSet>
 
+	<changeSet id="20240813-form-resource-description" author="loliveira">
```

Comment:
I think this needs to be retargeted at liquibase-update-to-latest-2.6.x.xml, no?